topics/sentinel.md

```diff
@@ -402,31 +405,66 @@ The following is a list of accepted commands:
 * **SENTINEL masters** Show a list of monitored masters and their state.
 * **SENTINEL master `<master name>`** Show the state and info of the specified master.
 * **SENTINEL slaves `<master name>`** Show a list of slaves for this master, and their state.
-* **SENTINEL get-master-addr-by-name `<master name>`** Return the ip and port number of the master with that name. If a failover is in progress or terminated successfully for this master it returns the address and port of the promoted slave.
+* **SENTINEL sentinels `<master name>`** Show a list of sentinels for this master, and their state.
+* **SENTINEL get-master-addr-by-name `<master name>`** Return the IP and port number of the master with that name. If a failover is in progress or terminated successfully for this master it returns the address and port of the promoted slave.
 * **SENTINEL reset `<pattern>`** This command will reset all the masters with matching name. The pattern argument is a glob-style pattern. The reset process clears any previous state in a master (including a failover in progress), and removes every slave and sentinel already discovered and associated with the master.
 * **SENTINEL failover `<master name>`** Force a failover as if the master was not reachable, and without asking for agreement to other Sentinels (however a new version of the configuration will be published so that the other Sentinels will update their configurations).
 
-Reconfiguring Sentinel at Runtime
+Reconfiguring Sentinel at Run-time
```

Comment:
Isn't "Runtime" more common?

Reply:
To add to the above, for the sake of consistency we could use `Runtime` I think, because we use `Runtime` / `runtime` in other places too in the same doc

Reply:
Just realized I'm commenting on a PR that's 7 years old :see